dashboard/app/assets/javascripts/angular/testExecutor.service.js

```diff
@@ -25,7 +25,7 @@
           method: 'DELETE'
         }
       });
-      var testExecUpdateIntervalPeriod = 5; // seconds
+      var testExecUpdateIntervalPeriod = 1; // seconds
```

Comment:
@redconfetti Its receiving a pass/fail/pending/skipped status for each step of each test.

Currently, the cucumber text representation of each step gets sent along with the status for the step. Since the cucumber text representation does not change over time, it only needs to be sent once and subsequent responses could contain just a sequence of step statuses. But switching to such an approach would require a significant amount of change, so am hoping to tackle at a future time.